src/Symfony/Component/HttpKernel/Attribute/MapQueryString.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Attribute;
+
+use Symfony\Component\HttpKernel\Controller\ArgumentResolver\RequestPayloadValueResolver;
+
+/**
+ * Controller parameter tag to map Query String to typed object and validate it.
+ *
+ * @author Konstantin Myakshin <molodchick@gmail.com>
+ */
+#[\Attribute(\Attribute::TARGET_PARAMETER)]
+final class MapQueryString extends ValueResolver
+{
+    public function __construct(
+        public readonly array $context = [],
+        bool $disabled = false,
```

Comment:
we can drop this because the resolver will be pinned so there is no point is disabling the behavior bound to the attribute (same for the other attribute)

Reply:
Fixed but what the reason to have this property at all? The only one attribute that can be disabled - `#[MapEntity]`

Reply:
or any resolver in the chain, with `#[ValueResolver(FooResolver::class, disabled: true)]`

Reply:
Ah, I wasn't aware that we have "universal" `#[ValueResolver]`